backend/src/database/events_attendance.go

```diff
@@ -232,3 +232,26 @@ func (db *DB) GetAttendanceFlagsForClass(classID int, args *models.QueryContext)
 
 	return flags, nil
 }
+
+func (db *DB) GetMissingAttendance(classID int, args *models.QueryContext) (int, error) {
+	var event models.ProgramClassEvent
+	err := db.WithContext(args.Ctx).
+		Preload("Class").
+		Where("class_id = ?", classID).
+		First(&event).Error
```

Comment:
@calisio Confirmed w/Rich that if we reschedule the series halfway through a class, we're giving the current program class event an UNTIL date and creating a new one going forward. That means we probably need to look past just the first event, right?